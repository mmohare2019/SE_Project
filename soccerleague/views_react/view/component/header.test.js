import "react-native";
import "react";
import Header from "./button";
import { test, expect } from "@jest/globals";
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test("Testing header component", function () {
    const component = renderer
    .create(<Header label={"Hello world"} />).toJSON();
    expect(component).toMatchSnapshot();
}); 