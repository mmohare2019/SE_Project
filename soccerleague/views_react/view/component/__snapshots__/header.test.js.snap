// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing header component 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#abd699",
      "borderColor": "#abd699",
      "borderRadius": 10,
      "borderWidth": 1,
      "marginLeft": 5,
      "marginRight": 5,
      "marginTop": 10,
      "opacity": 1,
      "paddingBottom": 10,
      "paddingTop": 10,
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 20,
        "paddingLeft": 10,
        "paddingRight": 10,
        "textAlign": "center",
      }
    }
  >
     
    Hello world
     
  </Text>
</View>
`;
